registry/Scripts/self-signed.ps1

```diff
@@ -1,13 +1,27 @@
 #requires -RunAsAdministrator
 
-# Password and Common Name of the certificate
-$PASSWORD=""
-$CN=""
-# Export path of the certificate
-$PATH=""
+Param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "Common DNS FQDN name to be used .")]
+    [string] $DNSCName,
+    [Parameter(Mandatory = $true, HelpMessage = "Certificate Password to be used to export the cert.")]
+    [string] $CertificateSecret,
+    [Parameter(Mandatory = $true, HelpMessage = "Certificate file export path where f.")]
+    [string] $CertificateFileExportPath
+)
+
+if (-not (Test-Path -Path $CertificateFileExportPath -IsValid))
+{
+    Write-Host "Error: Invlid path syntax for CertificateFileExportPath variable."
```

Comment:
typo `Invlid`
`CertificateFileExportPath is not a valid path` may be slightly better

Reply:
updated.